.github/workflows/master.yml

```diff
@@ -95,8 +95,8 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        stack: ["2.3.1"]
-        ghc: ["8.8.4"]
+        stack: ["2.7.3"]
+        ghc: ["8.10.4"]
```

Comment:
maybe we should just avoid setting the GHC version and remove `--system-ghc` in all the `stack` calls?

Reply:
I have no idea how that stack version of the CI workflow works, no idea what GHC version would be picked if we leave it out (if any). All I know is that we don't want to specify many of them since the cabal jobs already test several versions of GHC.

Reply:
The `ghc` version here just specifies what version of ghc will be installed globally. Calling `stack` with `--system-ghc` will tell it to find some `ghc` on `PATH` and use it.

If GHC weren't specified, `stack` would use the version of GHC in the resolver of the `stack.yaml` file. This would reduce the duplication between setting GHC version both in `stack.yaml` (via the resolver) and this line.